shared/wallets/common/account-entry/index.stories.js

```diff
@@ -10,7 +10,7 @@ const account = {
 }
 
 const load = () => {
-  Sb.storiesOf('Wallets/Common/Participants Row', module)
+  Sb.storiesOf('Wallets/Common/Account Entry', module)
```

Comment:
Think I might have been doing a little too much copy-pasting when I did this the first time